pkg/codegen/testing/test/testdata/output-funcs/python/pulumi_mypkg/func_with_empty_outputs.py

```diff
@@ -10,27 +10,11 @@
 from . import _utilities
 
 __all__ = [
-    'FuncWithEmptyOutputsResult',
-    'AwaitableFuncWithEmptyOutputsResult',
     'func_with_empty_outputs',
-    'func_with_empty_outputs_output',
 ]
 
-@pulumi.output_type
-class FuncWithEmptyOutputsResult:
-    def __init__(__self__):
-        pass
-
-class AwaitableFuncWithEmptyOutputsResult(FuncWithEmptyOutputsResult):
-    # pylint: disable=using-constant-test
-    def __await__(self):
-        if False:
-            yield self
-        return FuncWithEmptyOutputsResult()
-
-
 def func_with_empty_outputs(name: Optional[str] = None,
-                            opts: Optional[pulumi.InvokeOptions] = None) -> AwaitableFuncWithEmptyOutputsResult:
+                            opts: Optional[pulumi.InvokeOptions] = None):
```

Comment:
This needs to return some `Awaitable` or `pulumi.Output[T]`. This must be a function that exists for side effects, so we need to offer a way to indicate "after the side effect happens, do X".

Reply:
This is the non-output version of the function invoke. It should be `Awaitable[None]`?

Reply:
I think it should be `Awaitable[None]`, yeah.